config/speechConfig_example.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
```

Comment:
A more common convention for this is probably `speechConfig.example.js` or `speechConfig.js.example`.

Reply:
I like `speechConfig.js.example`